chain/network/src/raw.rs

```diff
@@ -343,3 +267,233 @@ impl Peer {
         Ok(())
     }
 }
+
+mod test {
+    use crate::network_protocol::{
+        Encoding, Handshake, PeerIdOrHash, PeerMessage, Pong, RawRoutedMessage, RoutedMessageBody,
+    };
+    use crate::raw;
+    use crate::time::Utc;
```

Comment:
Don't embed test module into the implementation. Create separate files:
raw/mod.rs
raw/tests.rs

Reply:
done